### ISSUE TITLE
Use GitHub Releases

### ISSUE THREAD

**Issue by schu** [kinvolk/stage1-builder#27]
Currently we rely on CircleCI assets for stage1-builder images:

```
<meta name="ac-discovery" content="kinvolk.io/aci/rkt/stage1-kvm-beta https://circleci.com/api/v1/project/kinvolk/stage1-builder/latest/artifacts/0/$CIRCLE_ARTIFACTS/stage1-kvm-{version}-linux-{kernelversion}.{ext}?filter=successful">
<meta name="ac-discovery" content="kinvolk.io/aci/rkt/stage1-kvm https://circleci.com/api/v1/project/kinvolk/stage1-builder/latest/artifacts/0/$CIRCLE_ARTIFACTS/stage1-kvm-{version}-linux-{kernelversion}.{ext}?filter=successful&branch=master">
```

This is brittle (images can change unexpectedly with newer stage1-builder versions; old images have to be rebuild on every stage1-builder update, as we point to `latest` artifacts; etc)

Instead, try to use GitHub Releases: https://help.github.com/articles/creating-releases/

**Comment by alban** [comment]
I agree. See also https://github.com/kinvolk/stage1-builder/pull/26#issuecomment-319055938

**Comment by alban** [comment]
Ideally we need to be able to both use release artifacts and artifacts from a specific build on a branch, in order to test changes in stage1-builder.

**Comment by lmb** [comment]
FWIW https://jedkirby.com/blog/github-releases-with-circleci has a tutorial for this. Maybe that is enough?
